src/openrct2-ui/windows/About.cpp

```diff
@@ -44,10 +44,20 @@ enum WINDOW_ABOUT_WIDGET_IDX {
     WIDX_MUSIC_CREDITS = WIDX_PAGE_START,
 };
 
+#ifndef LEFT_CLOSEBOX
+    // close button on right of window
+    #define CLOSEBOX_WIDGET \
+        { WWT_CLOSEBOX,         0,  WW - 13,        WW - 3,         2,          13,         STR_CLOSE_X,                            STR_CLOSE_WINDOW_TIP }
+#else
+    // close button on left of window
+    #define CLOSEBOX_WIDGET \
+        { WWT_CLOSEBOX,         0,  2,                  12,         2,          13,         STR_CLOSE_X,                            STR_CLOSE_WINDOW_TIP }
+#endif
+
 #define WIDGETS_MAIN \
       WWT_FRAME,            0,  0,              WW - 1,         0,          WH - 1,     0xFFFFFFFF,                             STR_NONE },             /* panel / background */ \
-    { WWT_CAPTION,          0,  1,              WW - 2,         1,          14,         STR_ABOUT,                              STR_WINDOW_TITLE_TIP }, /* title bar */ \
-    { WWT_CLOSEBOX,         0,  WW - 13,        WW - 3,         2,          13,         STR_CLOSE_X,                            STR_CLOSE_WINDOW_TIP }, /* close x button */ \
+    { WWT_CAPTION,          0,  1,              WW - 2,         1,          14,         STR_ABOUT,                              STR_WINDOW_TITLE_TIP }, \
+    CLOSEBOX_WIDGET, \
```

Comment:
I think it should be possible to replace the first three definitions of every window with a define that takes a window title, window height and window width. This define can then take left/right close button status into account.